x/ibc/03-connection/keeper/handshake.go

```diff
@@ -56,68 +56,57 @@ func (k Keeper) ConnOpenTry(
 	consensusHeight uint64,
 ) error {
 	// XXX: blocked by #5078
-	/*
-		if consensusHeight > uint64(ctx.BlockHeight()) {
-			return sdkerrors.Wrap(ibctypes.ErrInvalidHeight, "invalid consensus height")
-		}
+	// if consensusHeight > uint64(ctx.BlockHeight()) {
```

Comment:
What is this blocked by? https://github.com/cosmos/cosmos-sdk/issues/5078 is closed.

Reply:
blocked by #5475. Will update the comments